### PR TITLE
new blog post for divider tabs

### PR DESCRIPTION
**Pull request by JamieMatthews**
### Blog title
Too Many Tabs

### Blog description
Recommendation to manage all them tabs!

### How to test
- Run `jekyll serve --safe`.
- Goto http://127.0.0.1:4000/workflow/2015/03/03/divider-tabs/

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by andrew-c** [comment]
:+1:

**Comment by jackdcrawford** [comment]
I'll take editor 2

File: _posts/2015-03-03-divider-tabs.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- Deployments
+- Documents
+- Triage
+
+![dividerTabs]({{ site.baseurl }}assets/img/dividerTabs.png)
+
+This works for me currently but I can imagine in time, this will become bigger.
+
+Inside the email tab I have my GMail and our company intranet, so the bare basics.
+Inside Deployments, I have the likes of Github, Travis CI, TeamCity etc
```
Comment (jackdcrawford) [inline _posts/2015-03-03-divider-tabs.md]: The end of this line should probably read
> TeamCity, etc.

**Comment by jackdcrawford** [comment]
Nice post @JamieMatthews, looks good for me locally! :+1:

File: _posts/2015-03-03-divider-tabs.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+I then stumbled across [Divider Tabs](https://chrome.google.com/webstore/detail/divider-tabs/mpbblhoepkmpgalgppdoahgeoknmjgpc?hl=en), Chrome plugin. This has been a complete life saver since downloading.. I know it won't be to everyones taste but it works for me nonetheless. 
+
+**So what does it do?**
+Well for the sake of simplicity, it creates a divider tab which can store your 'favourites' or 'most used' links. You can add however many links into the divider tab and once you 'lock' the tab it saves those links until you decide to remove them or edit them.
+You are able to create as many as you want from what I can see, but I only require 4/5 'divider tabs' on a day to day basis.
+
+The way that I use this is to have categories for the links, just like you would in a file cabinet with different categories, except this is virtual. 
+The categories I have currently are;
```
Comment (jackdcrawford) [inline _posts/2015-03-03-divider-tabs.md]: This should be colon rather than a semi-colon

File: _data/members.yml [hunk 3]
```diff
@@ -67,4 +67,9 @@ rob_johnson:
   name: Rob Johnson
   role: Developer
   gravatar_md5: 4550a440411b13515e47f9ec14eddd21
-  twitter: notABluesSinger
\ No newline at end of file
+  twitter: notABluesSinger
+jamie_matthews:
```
Comment (jackdcrawford) [inline _data/members.yml]: We should try & keep this file in alphabetical order. Failing that we can look into doing it automatically :)

**Comment by jodiedoubleday** [comment]
For SEO, it would be great to make your blog URL the same name as the title of your article.

Also would be good (but not necessary) to make the first paragraph longer so that it looks better on the hungry geek homepage. (a summary to pull people in)

**Comment by JamieMatthews** [comment]
Made the changes you recommended

**Comment by jackdcrawford** [comment]
:+1: from me

**Pull request merged**